### ISSUE TITLE
backend_setup role fails with invalid argument for `lvcreate`

### ISSUE THREAD

**Issue by anoopcs9** [gluster/gluster-ansible-infra#129]
Following error is seen while preparing backend for GlusterFS cluster:
```
TASK [gluster.infra/roles/backend_setup : Create a LV thinpool for similar device types] ***
failed: [storage0] (item={'vgname': 'vg_vdb', 'thinpoolname': 'vg_vdb_thinpool', 'thinpoolsize': '9G', 'poolmetadatasize': '100M'}) => {"ansible_loop_var": "item", "changed": true, "cmd": ["lvcreate", "--type", "thin-pool", "--zero", "n", "-l", "9G", "--chunksize", "256K", "--poolmetadatasize", "100MiB", "-n", "vg_vdb_thinpool", "vg_vdb"], "delta": "0:00:00.012250", "end": "2022-06-02 10:02:10.161117", "item": {"poolmetadatasize": "100M", "thinpoolname": "vg_vdb_thinpool", "thinpoolsize": "9G", "vgname": "vg_vdb"}, "msg": "non-zero return code", "rc": 3, "start": "2022-06-02 10:02:10.148867", "stderr": "  Invalid argument for --extents: 9G\n  Error during parsing of command line.", "stderr_lines": ["  Invalid argument for --extents: 9G", "  Error during parsing of command line."], "stdout": "", "stdout_lines": []}
```
It looks like wrong option `-l` is used instead of `-L`.

**Comment by rchikatw** [comment]
Hello Anoop,

Looks like there is an issue with passing thinpoolsize parameter, As of now don't pass that value it will default available space.

I will commit the required changes within 1-2 days.

**Comment by anoopcs9** [comment]
Fixed with #130

**Comment by gobindadas** [comment]
PR merged, so closing this issue.
